MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -156,35 +158,51 @@ -(void) saveSpotifyData:(NSDictionary *) spotifyData{
     PFUser *current = [PFUser currentUser];
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     [query findObjectsInBackgroundWithBlock:^(NSArray *results, NSError *error) {
-        if (![[results valueForKey:@"userId"]containsObject:current.objectId]){
-            //Checks if User has already been added to database before adding
-            music[@"userId"] = current.objectId;
+        [User user].artists = [NSArray arrayWithArray:spotifyData[@"images"]];
+        if (![[[results valueForKey:@"author"] valueForKey:@"objectId"] containsObject: current.objectId]){
```

Comment:
I now check if the user matches the object ID instead of previously checking if the object ids of both were the same and adding the object Ids